Added checkSpam

EvgeniaKapuza opened the pull request:
проверна на спам

EvgeniaKapuza committed:
Added checkSpam

проверна на спам

jsru commented:
Добавляю преподавателя (@koalex) для код-ревью.

EvgeniaKapuza committed:
Merge branch 'master' of https://github.com/js-tasks-ru/jsbasic-20220310_evgenia-kapuza

jsru commented:
Проверьте, пожалуйста, ваше решение, не все тесты прошли (PR не будет принят до тех пор, пока **все** добавленные задачи не будут решены).

EvgeniaKapuza committed:
Merge branch 'master' of https://github.com/js-tasks-ru/jsbasic-20220310_evgenia-kapuza

EvgeniaKapuza committed:
Merge branch 'master' of https://github.com/js-tasks-ru/jsbasic-20220310_evgenia-kapuza

jsru commented:
Проверьте, пожалуйста, ваше решение, не все тесты прошли (PR не будет принят до тех пор, пока **все** добавленные задачи не будут решены).

jsru commented:
Проверьте, пожалуйста, ваше решение, не все тесты прошли (PR не будет принят до тех пор, пока **все** добавленные задачи не будут решены).

EvgeniaKapuza committed:
Added... factorial,function isValid,ucFirst(str)

Вычислила факториал
Допиcала функцию валидации имени пользователя
Сделала первый символ заглавным

jsru commented:
Проверьте, пожалуйста, ваше решение, не все тесты прошли (PR не будет принят до тех пор, пока **все** добавленные задачи не будут решены).

EvgeniaKapuza committed:
Added function truncate(str, maxlength)

усечение строки

jsru commented:
Решение было обновлено, посмотрим что скажет @koalex
